src/ripple/overlay/impl/ConnectAttempt.cpp

```diff
@@ -317,7 +317,7 @@ ConnectAttempt::processResponse()
             if (json.isObject() && json.isMember("peer-ips"))
             {
                 Json::Value const& ips = json["peer-ips"];
-                if (ips.isArray())
+                if (ips.isArrayOrNull())
```

Comment:
`isArray` is sufficient here. If it's a null object there's no real reason to do any work.

Reply:
Fixed.